deployment/utils/operations/contract/write.go

```diff
@@ -0,0 +1,66 @@
+package contract
```

Comment:
TODO: Could probably just move this one back to EVM, since sequence outputs are aligned with mcms_types.BatchOperation